### DISCUSSION TITLE
我的路由器是MI-R3P，应该如何安装？

### DISCUSSION THREAD

**Discussion by magellan-99** [messense/aliyundrive-webdav#552] [Q&A] [answered]
[MI-R3P /tmp]# opkg print-architecture
arch all 100
arch mipsel-3x 150
arch mipsel-3.4 160
如题，的确是小白，不懂

**Comment by prunes81485216** [comment, marked as answer]
aliyundrive-webdav_1.8.9-1_mipsel_24kc.ipk
luci-app-aliyundrive-webdav_1.8.9_all.ipk
luci-i18n-aliyundrive-webdav-zh-cn_1.8.9-1_all.ipk
请下载安装以上对应的三个即可

**Reply by messense** [reply]
如果还有问题，可以在 `/etc/opkg.conf` 中增加如下配置

```
arch all 100
arch mipsel-3x 150
arch mipsel-3.4 160
arch mipsel_24kc 200
```
